app/javascript/dashboard/components/widgets/conversation/conversationCardComponents/CardLabels.vue

```diff
@@ -2,10 +2,11 @@
   <div
     v-show="activeLabels.length"
     ref="labelContainer"
```

Comment:
Let's remove this ref if it's not used

Reply:
The `labelContainer` ref is used in the method `computeVisibleLabelPosition`.